src/ui.cpp

```diff
@@ -778,9 +779,13 @@ void uilist::show( ui_adaptor &ui )
 
             mvwprintz( window, point( pad_left + 1, estart + si ), co, padspaces );
             if( entries[ei].hotkey.has_value() && entries[ei].hotkey.value() != input_event() ) {
-                const nc_color hotkey_co = ei == selected ? hilight_color : hotkey_color;
-                mvwprintz( window, point( pad_left + 1, estart + si ), entries[ ei ].enabled ? hotkey_co : co,
-                           "%s", right_justify( entries[ei].hotkey.value().short_description(), 2 ) );
```

Comment:
The hotkey description could be two letters long in the keycode mode (for example ctrl+a will be displayed as `^a`):

https://github.com/CleverRaven/Cataclysm-DDA/blob/6d311a79b1ba922a0bd86282b3f2290e47a81428/src/input.cpp#L159-L163

So you should keep `right_justify` and add one more to pad_left in line 797 below.

Reply:
I think this did the trick https://github.com/CleverRaven/Cataclysm-DDA/pull/64478/commits/cdf95fbd01f4cab869bb3ccf1b10cd97e65acadd

Reply:
I did my best to mark which changes are probably not needed, but the uilist code is a mess and I'm tired analyzing magic number now. Might want to fine-tune it later instead.

Reply:
Thanks for being so meticulous. This https://github.com/CleverRaven/Cataclysm-DDA/pull/64478/commits/2fbc662ed6e32eee9b009acc12f23fd661beb2c0 should resolve the issue. I replaced my WIDTH_BUFFER variable back to 4 everywhere one-by-one, recompiled and tested a few uilists until I found which ones needed to actually be 6 now :) due to the added padding